src/middleware.js

```diff
@@ -57,9 +57,13 @@ const actionsWithTags = new Set([
  */
 const requestErrorActionsMap = {
   [LOAD_REQUEST]: LOAD_ERROR,
+  [LOAD_ERROR]: LOAD_ERROR,
```

Comment:
Can you just document when we have `LOAD_REQUEST` with flag and when we have `LOAD_ERROR`. It is a little bit confusing on the first.